docs/user/monitoring/viewing-metrics.asciidoc

```diff
@@ -13,6 +13,15 @@ At a minimum, you must have monitoring data for the {es} production cluster.
 Once that data exists, {kib} can display monitoring data for other products in
 the cluster.
 
+TIP: If you use a separate monitoring cluster to store the monitoring data, it
+is strongly recommended that you use a separate {kib} instance to view it. If
+you use SAML, Kerberos, PKI, or OpenID Connect realms on your production or
```

Comment:
__note:__ Sorry for not mentioning it earlier, but I think we also want to mention that users will also have to use a dedicated {kib} instance if Kibana is configured to use [Token authentication provider](https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#token-authentication) (it may rely on {es} Native, Reserved, LDAP, AD or any other custom password-based realm under the hood). This provider isn't widely used right now, but it's the only option if users want to authenticate with LDAP/AD where OTP is a requirement and we eventually want to make it a default provider instead of [Basic](https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#basic-authentication) as well.

Something similar to this, but feel free to rephrase:
 ```suggestion
you log in to {kib} using SAML, Kerberos, PKI, OpenID Connect, or Token authentication provider on your production or
```

Reply:
Thanks for the information!  I've added the suggested text.